var/spack/repos/builtin/packages/bref3/package.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+import os.path
+
+
+class Bref3(Package):
+    """Bref3: Converts from VCF format to bref3 format."""
+
+    homepage = "http://faculty.washington.edu/browning/beagle/beagle.html"
+
+    version('25Nov19', sha256='969c0881050c4a48d19be9ea64bf49fa68c1403b69f9f739bbfd865dda639b2d', 
```

Comment:
These versions won't sort correctly, I would use:
```suggestion
    version('2019-11-25', sha256='969c0881050c4a48d19be9ea64bf49fa68c1403b69f9f739bbfd865dda639b2d', 
```
Then `preferred=True` won't be needed anymore.